docs/user/algo_bc.md

```diff
@@ -0,0 +1,16 @@
+# Behavioral Cloning
```

Comment:
I don't see your plots here?

Reply:
sorry -- by this i meant "are there any plots from baseline experiments you can include?"